### DISCUSSION TITLE
Data XML file

### DISCUSSION THREAD

**Discussion by BoschHU** [nav-gov-hu/eVAT#46] [Általános]
Az eÁfa M2M_Folyamat_GitHub. pdf leírásban szó van a data XML struktúráról, mely a bevallás sorait tartalmazza, a dokumentumban jelenleg képként beszúrva. Meg lehetne kapni a data XML file-t xls formátumban? Benne szeretnénk kérni az opcionális és kötelező mezők egyértelmű jelölését. Köszönjük.

**Comment by BoschHU** [comment]
Kiegészíteném, hogy a data XML struktúra elemeinek speciális tulajdonságai is hasznosak lennének, például van-e karakterlimit, dátumokat milyen formátumban kell megadni, stb.

**Comment by renced42** [comment]
Kedves @BoschHU 

Az XML leíró XSD-t a https://github.com/nav-gov-hu/eVAT/tree/master/src/schemas/hu/gov/nav/vdr alatt lehet megtalálni.
